test/views/optOut/ConfirmOptOutViewSpec.scala

```diff
@@ -90,5 +90,9 @@ class ConfirmOptOutViewSpec extends TestSupport {
       pageDocument.getElementById("cancel-button").text() shouldBe confirmOptOutMessages.cancelButton
       pageDocument.getElementById("confirm-optout-form").attr("action") shouldBe confirmOptOutMessages.confirmedOptOutURLAgent
     }
+
+    "with quarterly updates as zero count" in new Setup(false) {
+      pageDocument.getElementById("warning-inset") shouldBe null
```

Comment:
should fix: lets not use `null`
substitute it with
```scala
pageDocument.select("#warning-inset").size() shouldBe 0
```

Reply:
done